documentation/book/proc-deleting-a-topic.adoc

```diff
@@ -12,6 +12,9 @@ This procedure describes how to delete a Kafka topic using a `KafkaTopic` {Produ
 * A running Kafka cluster.
 * A running Topic Operator.
 * An existing `KafkaTopic` to be deleted.
+* `delete.topic.enable=true`
+
+NOTE: The `delete.topic.enable` property must be set to `true` in `Kafka.spec.kafka.config`. Otherwise, the steps outlined here will delete the `KafkaTopic` resource, but the Kafka topic and its data will remain. After reconciliation by the Cluster Operator, the custom resource is then recreated.
```

Comment:
in this case it's the Topic Operator to reconcile and create the `KafkaTopic` resource not the CO. Anyway `delete.topic.enable` is set to true by default in Kafka.